gravity.sh

```diff
@@ -538,7 +536,7 @@ gravity_ParseUserDomains() {
     return 0
   fi
 
-  gravity_ParseDomainsIntoHosts "${blacklistFile}" "${blackList}.tmp"
+  mv "${blacklistFile}" "${blackList}.tmp"
```

Comment:
Blacklisting is not working properly.

This now moves `blacklist.txt` to `blacklist.tmp`, and then `blacklist.tmp` to `black.list`.

`pihole -b` is now broken!